docs/Contributing/architecture/system_design/cache.md

```diff
@@ -0,0 +1,68 @@
+[Back to top](./README.md)
+# Cache
+
+Fleet uses Redis (standalone or cluster) as an ephemeral coordination and acceleration layer. No authoritative data stored here—safe to flush with minor transient impact.
+
+## Use cases
+* Live query campaigns (active query set + host completion tracking)
+* Distributed locks for cron / async tasks (`redis_lock` namespace)
```

Comment:
Not sure this is correct, I think cron uses mysql for locks? It does exist though, looks to be used for the google calendar feature?